UPGRADE.md

```diff
@@ -7,6 +7,13 @@
 Write permissions for the webserver must be set on `web/uploads` instead of
 `web/uploads/media` alone to support simple cache clearing.
 
+### BlameSubscriber
+
+The BlameBehavior has been moved from the DocumentManager component to the
+Sulu Content component. Documents which implemented
+`Sulu\Component\DocumentManager\Behavior\Audit\BlameBehavior` should now
```

Comment:
is this deprecated now? the interface could be there (deprecated) and extends the new one.